library.gradle

```diff
@@ -10,7 +10,7 @@ android {
         minSdkVersion minVersion
         targetSdkVersion compileVersion
         versionCode 1
-        versionName "4.2.0"
+        versionName "4.2.0-SNAPSHOT"
```

Comment:
Should be `4.3.0-SNAPSHOT`. Can you also add it to `Config.kt` and `gradle.properties`. Thanks!